### DISCUSSION TITLE
How to do a big-time auto format in place?

### DISCUSSION THREAD

**Discussion by fmv1992** [marlonrichert/zsh-hist#19] [Q&A] [answered]
Hi, in a no brainer way, how can I reformat my ~150k lines long `$HOME/.zsh_history` in place with its history auto formatted?

I cannot figure out how to use your plugin in such a scale (I managed to get a few lines formatted).

**Comment by marlonrichert** [comment, marked as answer]
`hist n \*` should do the trick.

**Reply by fmv1992** [reply]
Thanks, it worked! I tried `hist '.*'` before.

Much appreciated!

**Reply by marlonrichert** [reply]
Yeah, the patterns are not regex; they are [glob patterns](https://zsh.sourceforge.io/Doc/Release/Expansion.html#Glob-Operators).
